### PR TITLE
added lowercase letters

### LINKED ISSUE #492: lowercase letters
could you add lowercase letters? you probably cant because its only 8x8 pixels. but still, could you?

### PR DESCRIPTION
**Pull request by KalmeMarq**
I also added ./run folder to the .gitignore. I use it as savedir in dev environment.
Resolves #492

### CODE REVIEW

File: src/client/java/minicraft/screen/Menu.java [hunk 1]
```diff
@@ -218,8 +218,8 @@ public void tick(InputHandler input) {
 					for (int i = 0; entryIt.hasNext(); i++) {
 						ListEntry entry = entryIt.next();
 
-						String stringEntry = entry.toString().toLowerCase(Locale.ENGLISH);
-						String typingString = typingSearcher.toLowerCase(Locale.ENGLISH);
+						String stringEntry = entry.toString();
+						String typingString = typingSearcher;
```
Comment (BenCheung0422) [inline src/client/java/minicraft/screen/Menu.java:222]: Entry searching should be case-insensitive.

**Review by BenCheung0422 — CHANGES_REQUESTED** [review]
The overall pull request is good, except that there are still some problems.

**Review by BenCheung0422 — DISMISSED** [review]

**Review by Makkkkus — CHANGES_REQUESTED** [review]
Make sure to change all the localisations keys for all languages.

I also don't see where the run directory is used.

**Review by ChristofferHolmesland — APPROVED** [review]

**Comment by Makkkkus** [comment]
> Make sure to change all the localisations keys for all languages.

Still waiting for you to fix this.

**Comment by BenCheung0422** [comment]
All changes should have already been applied on all languages.

File: src/client/resources/assets/localization/en-us.json [hunk 2]
```diff
@@ -50,8 +50,8 @@
   "minicraft.display.options_display.change_key_bindings": "Change Key Bindings",
   "minicraft.display.options_display.language": "Language",
   "minicraft.display.options_display.resource_packs": "Resource packs",
-  "minicraft.display.popup.enter_confirm": "enter to confirm",
-  "minicraft.display.popup.escape_cancel": "escape to cancel",
+  "minicraft.display.popup.enter_confirm": "Enter to confirm",
+  "minicraft.display.popup.escape_cancel": "Escape to cancel",
```
Comment (Makkkkus) [inline src/client/resources/assets/localization/en-us.json:54]: What I'm referring to.

Comment (BenCheung0422) [reply src/client/resources/assets/localization/en-us.json:54]: I understand now. The change needs to be reflected in only nb-no and ru-ru while other localizations are not done as in files yet.

Comment (KalmeMarq) [reply src/client/resources/assets/localization/en-us.json:54]: Sorry for taking a while.
So what I need to do is capitalize this keys in every language or reverted? Sorry for not understanding immediately.

Comment (BenCheung0422) [reply src/client/resources/assets/localization/en-us.json:54]: Only 2 languages have these keys in files for now. This kind of string should always be capitalized as they occupy the whole entry.

Comment (Makkkkus) [reply src/client/resources/assets/localization/en-us.json:54]: Yeah, if you cange one language, change them all. You don't need to know the other languages to change the capitalisation.

**Review by BenCheung0422 — APPROVED** [review]

**Review by Makkkkus — APPROVED** [review]

**Pull request merged**